src/fpylll/fplll/decl.pxd

```diff
@@ -228,12 +228,12 @@ ELSE:
             FastEvaluator[FP_NR[dpe_t]] *dpe
             FastEvaluator[FP_NR[dd_real]] *dd
             FastEvaluator[FP_NR[qd_real]] *qd
-            FastEvaluator[FP_NR[mpfr_t]] *mpfr
+            FastErrorBoundedEvaluator *mpfr
     ELSE:
         ctypedef union fast_evaluator_core_t:
             FastEvaluator[FP_NR[double]] *double
             FastEvaluator[FP_NR[dpe_t]] *dpe
-            FastEvaluator[FP_NR[mpfr_t]] *mpfr
+            FastErrorBoundedEvaluator *mpfr
```

Comment:
I don't know, but it's the same change that was made in 3ca12c7528b5cfebd34d2421b119133b4f03942c, but just wasn't propagated to this branch if the IF/ELSE